test/lib/commands/shrinkwrap.js

```diff
@@ -37,13 +37,13 @@ const shrinkwrap = async (t, prefixDir = {}, config = {}, mocks = {}) => {
   const oldFile = resolve(npm.prefix, 'package-lock.json')
 
   t.notOk(fs.existsSync(oldFile), 'package-lock is always deleted')
-  t.same(logs.warn, [], 'no warnings')
+  // t.same(logs.warn, [], 'no warnings')
```

Comment:
why did we have to remove this?

Reply:
Good catch. This needs investigation.

Reply:
Another case of old logs not being mocked properly. The warning events within arborist wern't being listened to so logs.warn was always empty. This now properly matches what users see.